eng/pipelines/libraries/base-job.yml

```diff
@@ -75,6 +75,10 @@ jobs:
           - _finalFrameworkArg: -allConfigurations
           - _extraHelixArguments: /p:BuildAllConfigurations=true
 
+        - ${{ if eq(parameters.isSourceBuild, true) }}:
+          - _finalFrameworkArg: /p:DotnetBuildFromSource=true
+          - _extraHelixArguments: /p:DotnetBuildFromSource=true
```

Comment:
We don't need to set this property right? `_extraHelixArguments` just comes into play when running tests